.github/workflows/test.yml

```diff
@@ -40,6 +40,6 @@ jobs:
           python -m pip install -U pip setuptools>=18.5
           python -m pip install -r requirements-dev.txt
 
-      - name: Tests
+      - name: Run tests, linting, and everything else
         shell: bash
-        run: ./scripts/run_tests.sh
+        run: tox
```

Comment:
Switching it to be tox-based where tox-gh-action will run the various tox environments depending on the Python version. Now we're running all the tox environments. Previously, we were only running the basic ones and skipping the tinycss2 ones and the build-no-lang ones in CI.